maven-core/src/main/java/org/apache/maven/project/DefaultProjectBuilder.java

```diff
@@ -115,14 +118,21 @@
     public ProjectBuildingResult build( File pomFile, ProjectBuildingRequest request )
         throws ProjectBuildingException
     {
-        return build( pomFile, new FileModelSource( pomFile ), new InternalConfig( request, null ) );
+        return build( pomFile, new FileModelSource( pomFile ),
+                new InternalConfig( request, null, useGlobalModelCache() ? getModelCache() : null ) );
+    }
+
+    private boolean useGlobalModelCache()
+    {
+        return !Boolean.parseBoolean( System.getProperty( DISABLE_GLOBAL_MODEL_CACHE_SYSTEM_PROPERTY ) );
```

Comment:
Why don't you just use `Boolean#getBoolean()`?

Reply:
Right, I'll change that.